src/runtime/runtimeClient.ts

```diff
@@ -94,6 +95,31 @@ export abstract class RuntimeClient {
     return true;
   }
 
+  enabledCss() {
+      var css = "";
+      this.lastParse.code.replace(/(?:```|~~~)css\n([\w\W]*?)\n(?:```|~~~)/g, (g0, g1) => { // \n excludes disabled blocks
+        css += g1;
+      });
+
+      // remove whitespace before open braces, and add a newline after open brace
+      css = css.replace(/\s*{\s*/g, " {\n");
+
+      css = css.split("\n").map(function(line) {
+        var trimmedLine = line.trim();
+        if ((line.indexOf("{") !== -1) && // selector line
+          ("0123456789@".indexOf(trimmedLine[0]) === -1) && // no animation percentages or @-rules
```

Comment:
I only bail here if the selector starts with a number (checking first character only), which selector grammar doesn't allow - so I think it should just exclude animation percentage specifications and not other selectors.